packages/page-accounts/src/modals/Qr.tsx

```diff
@@ -70,6 +71,18 @@ function QrModal ({ className = '', onClose, onStatusChange }: Props): React.Rea
     [_onNameChange]
   );
 
+  const _onError = useCallback(
+    (err: Error): void => {
+      setAddress({
+        address: '',
+        isAddress: false,
+        scanned: null,
+        warning: t<string>(err.message)
```

Comment:
Small comment, since this is an error (non-translatable), should just be `err.message`